build/mac/BUILD.gn

```diff
@@ -6,22 +6,24 @@ import("//build/util/version.gni")
 import("//chrome/common/features.gni")
 import("//third_party/widevine/cdm/widevine.gni")
 
+mac_unsigned_output_prefix = "unsigned"
+unsigned_app_path = "$root_out_dir/$brave_exe"
+unsigned_dmg_path = "$root_out_dir/$mac_unsigned_output_prefix/$brave_dmg"
+unsigned_pkg_path = "$root_out_dir/$mac_unsigned_output_prefix/$brave_pkg"
+
 mac_signed_output_prefix = "signed"
+signed_dmg_path = "$root_out_dir/$mac_signed_output_prefix/$brave_dmg"
+signed_pkg_path = "$root_out_dir/$mac_signed_output_prefix/$brave_pkg"
+
 mac_notarized_output_prefix = "notarized"
+notarized_dmg_path = "$root_out_dir/$mac_notarized_output_prefix/$brave_dmg"
+notarized_pkg_path = "$root_out_dir/$mac_notarized_output_prefix/$brave_pkg"
+
 notary_asc_provider = chrome_mac_team_id
-unsigned_app_path = "$root_out_dir/$brave_exe"
-packaging_dir = "$brave_packaging_base_dir/" +
-                string_replace("$chrome_product_full_name Packaging", " ", "_")
-keychain_db =
-    getenv("HOME") + "/Library/Keychains/${mac_signing_keychain}.keychain-db"
-unsigned_pkg_path = "$root_out_dir/unsigned/$brave_pkg"
 
-if (is_universal_binary) {
-  x64_app_path_prefix = "x64"
-  arm64_arch_app_path = unsigned_app_path
-  x64_arch_app_path = "$root_out_dir/$x64_app_path_prefix/$brave_exe"
-  unsigned_app_path = "$brave_packaging_base_dir/$brave_exe"
-}
+packaging_dir = "$brave_packaging_base_dir/" + string_replace("$chrome_product_full_name Packaging", " ", "_")
+
+keychain_db = getenv("HOME") + "/Library/Keychains/${mac_signing_keychain}.keychain-db"
```

Comment:
I know this is existing code that was moved, but this should really be a gn config var that we pass in. Maybe a follow-up to fix it?

Reply:
have moved to `build/mac/config.gni` if that's what you meant